ios/RNModules/RNLogging/RNLogging/RNLoggingModule.m

```diff
@@ -45,29 +31,61 @@ + (BOOL)requiresMainQueueSetup {
     return YES;
 }
 
-- (id<RNLogger>)logger {
-    if (!_logger) {
-        return [[RNDummyLogger alloc] init];
-    }
-    return _logger;
-}
-
-RCT_EXPORT_METHOD(ancillaryDisplayed:(NSString *)type  provider:(NSString *)provider) {
+RCT_EXPORT_METHOD(ancillaryDisplayed:(NSString *)type provider:(NSString *)provider) {
     [self.logger ancillaryDisplayed:type provider:provider];
 }
 
 RCT_EXPORT_METHOD(ancillaryPurchased:(NSString *)type provider:(NSString *)provider) {
     [self.logger ancillaryPurchased:type provider:provider];
 }
 
+RCT_EXPORT_METHOD(hotelsResultsDisplayed:(id)additionalInfo) {
```

Comment:
From the logging spreadsheet, it seems like the arguments should be:
searchQuery: nullable string
params: nullable string

Reply:
Changed.